File: 4st/Hani/programmer/짝수 홀수 개수.java
class Solution {
    public int[] solution(int[] num_list) {
        int[] answer = new int[2];
        for (int i = 0; i < num_list.length; i++) {
            if (num_list[i] % 2 == 0)
                answer[0] += 1;
            else
                answer[1] += 1;
        }
        return answer;
    }
}

File: 4st/Hani/programmer/짝수 홀수 개수.py
def solution(num_list):
    answer = [0, 0]
    for i in range(len(num_list)) :
        if num_list[i] % 2 == 0 :
            answer[0] += 1
        elif num_list[i] % 2 == 1 :
            answer[1] += 1
    return answer